src/Controls/tests/DeviceTests/Elements/SearchBar/SearchBarTests.cs

```diff
@@ -54,5 +71,32 @@ public async Task MaxLengthIsReadOnlyValueTest()
 			});
 		}
 #endif
+
+#if ANDROID
+		[Fact]
+		public async Task SearchBarTakesFullWidth()
+		{
+			SetupBuilder();
+
+			SearchBar searchBar = new();
+
+			ContentPage page = new()
+			{
+				Content = searchBar
+			};
+
+			NavigationPage navPage = new(new ContentPage());
```

Comment:
Instead of using page specifically, I wonder if we can just compare to whatever the parent container is in general?